libraries/DataFlash/DataFlash_File.cpp

```diff
@@ -182,6 +183,18 @@ bool DataFlash_File::log_exists(const uint16_t lognum) const
     return ret;
 }
 
+void DataFlash_File::periodic_1Hz(const uint32_t now)
+{
+    if (!io_thread_alive()) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "IO thread dead");
```

Comment:
Can we have a better text message? This message isn't really informative. For a user IO might mean RC IO and he'll just freak out.